python/kfserving/kfserving/handlers/http.py

```diff
@@ -43,21 +47,46 @@ def validate(self, request):
             )
         return request
 
+    def has_ce_headers(self, headers: typing.Dict[str, str]) -> bool:
+        return (
+            "ce-specversion" in headers
+            and "ce-source" in headers
+            and "ce-type" in headers
+            and "ce-id" in headers
+        )
```

Comment:
could we call=```has_binary_headers``` https://github.com/cloudevents/sdk-python/blob/master/cloudevents/sdk/converters/util.py#L4

```from cloudevents.sdk.converters.util import has_binary_headers```

Reply:
Thanks @wengyao04 for pointing this out. 
Have updated now.